plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -504,14 +504,6 @@ func GetResourceRequest(pod *v1.Pod) *schedulercache.Resource {
 		result.Add(container.Resources.Requests)
 	}
 
-	// Account for storage requested by emptydir volumes
-	// If the storage medium is memory, should exclude the size
-	for _, vol := range pod.Spec.Volumes {
-		if vol.EmptyDir != nil && vol.EmptyDir.Medium != v1.StorageMediumMemory {
-			result.StorageScratch += vol.EmptyDir.SizeLimit.Value()
```

Comment:
does this mean the SizeLimit field is getting dropped?

Reply:
No, one single local ephemeral storage can be set in container's resources,
```
 containers:
   - name: fooa
     resources:
        requests:
  	 ephemeral-storage: “10Gi”
        limits:
  	 ephemeral-storage: “10Gi”
```
predicate functions take that into account and do not consider Emptydir any more.
btw: EmptyDir's sizelimit will be used by eviction manager